app/main/views/marketplace.py

```diff
@@ -118,7 +116,11 @@ def list_opportunities(framework_framework):
 
     api_result = form.get_briefs()
 
-    briefs = api_result["briefs"]
+    briefs = [{
```

Comment:
It feels weird to have to pass through both the raw data and content to the template.  Does the content alone not have everything we need to display the brief?  And if not, can we make the content have everything we need to display the brief?